### PR TITLE
Readme Bump

### PR DESCRIPTION
**Pull request by lazypower**
Moves and updates the Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+`layer.yaml`
+
+```yaml
+includes:
+  - beats-base
+```
+
+`metadata.yaml`
+
+```
```
Comment (mbruzek) [inline README.md:44]: /```yaml

File: README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+@when('elasticsearch.available')
+@when_not('mybeat.index.pushed')
+def push_packetbeat_index(elasticsearch):
+    hosts = elasticsearch.list_unit_data()
+    for host in hosts:
+        host_string = "{}:{}".format(host['host'], host['port'])
+    push_beat_index(host_string, 'mybeat')
+    set_state('mybeat.index.pushed')
+
+```
```
Comment (mbruzek) [inline README.md]: Does this layer set any states that others can react to? Or is it simply a layer that contains a library?

**Comment by AdamIsrael** [comment]
Looks good to me! :thumbsup:

**Comment by lazypower** [comment]
I'll squash this before merging if you want. Still looking for some terms breakage here. I think i left in 'chuckisms' and haven't fully leveraged the juju language

**Comment by mbruzek** [comment]
+1

**Pull request merged**